lib/active_shipping/carriers/fedex.rb

```diff
@@ -472,10 +472,15 @@ def parse_rate_response(origin, destination, packages, response, options)
 
       success = response_success?(xml)
       message = response_message(xml)
+      missing_xml_field = false
 
       if success
         rate_estimates = xml.root.css('> RateReplyDetails').map do |rated_shipment|
-          service_code = rated_shipment.at('ServiceType').text
+          service_code = rated_shipment.at('ServiceType').try(:text)
+          unless service_code.present?
+            missing_xml_field = true
+            next
+          end
```

Comment:
Since this is a change in the API, I see that other errors could happen later on in the process, such as `rated_shipment.at('RatedShipmentDetails/ShipmentRateDetail/TotalNetCharge/Currency').text`. Since we probably don't want to wrap all the calls with a `if` like the above, should we simply rescue `NoMethodError`, in which we could log?